.github/workflows/test.yml

```diff
@@ -178,38 +178,30 @@ jobs:
       - name: Setup PostgreSQL root user
         run: sudo --user=postgres createuser --superuser root
 
-      - name: Build local debian package
+      - name: Build package
         uses: linz/linz-software-repository@v9
         with:
           release: ${{ steps.distro.outputs.codename }}
           packages: jq
-        env:
-          PG_SUPPORTED_VERSIONS: ${{ matrix.pg }}
 
       - name: Install postgresql-agnostic tableversion package
-        run: sudo dpkg -i build-area/tableversion_*.deb
```

Comment:
Is this just so it is more readable? Or something else?

Reply:
It's more readable. Anyone not familiar with the `dpkg` command structure can *guess*, but it's about minimising the cognitive overhead.